src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -142,26 +169,37 @@ private enum Flags : byte
                 _name = _lazySourceName = memberName;
             }
 
-            if ((isAutoProperty && hasGetAccessor) || hasInitializer)
+            if (isAutoProperty || hasInitializer)
             {
                 Debug.Assert(!IsIndexer);
-                string fieldName = GeneratedNames.MakeBackingFieldName(_name);
-                BackingField = new SynthesizedBackingFieldSymbol(this,
-                                                                      fieldName,
-                                                                      isReadOnly: (hasGetAccessor && !hasSetAccessor) || isInitOnly,
-                                                                      this.IsStatic,
-                                                                      hasInitializer);
+                _ = CreateBackingField(isCreatedForFieldKeyword: hasInitializer && !isAutoProperty);
             }
 
             if (hasGetAccessor)
             {
-                _getMethod = CreateGetAccessorSymbol(isAutoPropertyAccessor: isAutoProperty, diagnostics);
+                _getMethod = CreateGetAccessorSymbol(isAutoPropertyAccessor: isAutoProperty && !getHasImplementation, diagnostics);
```

Comment:
> && !getHasImplementation [](http://example.com/codeflow?start=91&length=25)

Is addition of this condition makes any difference given the current state of the PR (commit 40)?

Reply:
@AlekseyTs Yes. One of the things it affects is whether compiler generated attribute is emitted or not.

But it's mostly for the "mixed" case `{ get; set => field = value }` that we are now not considering for this PR.